homeassistant/components/sms/sensor.py

```diff
@@ -0,0 +1,78 @@
+"""Support for SMS dongle sensor."""
+import logging
+
+import gammu  # pylint: disable=import-error, no-member
+
+from homeassistant.const import DEVICE_CLASS_SIGNAL_STRENGTH
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN, SMS_GATEWAY, SMS_GSM_SIGNAL_SENSOR_NAME
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the GSM Signal Sensor sensor."""
+    name = SMS_GSM_SIGNAL_SENSOR_NAME
+    gateway = hass.data[DOMAIN][SMS_GATEWAY]
+    entities = []
+    entities.append(GSMSignalSensor(hass, gateway, name,))
+    async_add_entities(entities, True)
+
+
+class GSMSignalSensor(Entity):
```

Comment:
These kind of sensors tend to be very spammy, and generally don't have too much added value. I suggest you disable it by default by implementing the `entity_registry_enabled_default` property and defaulting it to `False`

Reply:
👍